docs/source/debugging.rst

```diff
@@ -0,0 +1,50 @@
+Debugging Errors
+================
+
+First, thank you for using Starfish and SpaceTx-Format! Feedback you provide on features and the
+user experience is critical to making Starfish a successful tool. Because we iterate quickly on this
+feedback to add new features, things change often, which can result in your code getting out of sync
+with your data. When that happens, you may observe errors.
+
+Most of the time, you can fix this problem by pulling the most recent version of the code,
+reinstalling starfish, and restarting your environment. If you're using starfish with datasets from
+spaceTx located on our cloudfront distribution, we're committed to keeping that data up to date.
+You can find it in a versioned folder, and the version will correspond to the version of starfish
+that it matches.
+
+If you're using your own data with starfish, you may need to re-run your data ingestion workflow
+based on TileFetcher and FetchedTile to generate up-to-date versions of spaceTx-format.
```

Comment:
link to docs?

Reply:
Added forward-facing references for when this is in the API.